Sources/MapboxNavigation/NavigationViewController.swift

```diff
@@ -110,6 +110,11 @@ open class NavigationViewController: UIViewController, NavigationStatusPresenter
         }
     }
     
+    /**
+     Controls whether dark style will be used whenever traversing through a tunnel. Defaults to `true`.
+     */
+    public var usesDarkStyleWhileInTunnel: Bool = true
```

Comment:
`usesNightStyleWhileInTunnel` would be consistent with the naming of NightStyle. The style’s name is counterintuitive, but it was based on the map style’s name and was implemented before we generalized the feature to tunnels. At least developers wouldn’t wonder if there’s a difference between a dark style and a night style.